server/db/index.js

```diff
@@ -1,5 +1,5 @@
 var pg = require('pg')
-pg.defaults.ssl = true
+pg.defaults.ssl = false
```

Comment:
This should remain as `true`, because we need ssl in production.
Can we move this in the configuration file and read the appropriate setting for `prod` and `dev`?